samples/boards/stm32/ccm/README.rst

```diff
@@ -1,7 +1,7 @@
 .. _olimex_stm32_e407_ccm:
 
-Olimex STM32-E407 CCM example
```

Comment:
this sample looks in fact generic enough, as soon as the board has CCM, right?

Reply:
Indeed. Removed reference to Olimex in header.